x/concentrated-liquidity/export_test.go

```diff
@@ -227,7 +227,7 @@ func (k Keeper) CollectIncentives(ctx sdk.Context, owner sdk.AccAddress, positio
 	return k.collectIncentives(ctx, owner, positionId)
 }
 
-func GetUptimeTrackerValues(uptimeTrackers []model.UptimeTracker) []sdk.DecCoins {
+func (k Keeper) GetUptimeTrackerValues(uptimeTrackers []model.UptimeTracker) []sdk.DecCoins {
```

Comment:
What is the reason for adding this? Can this be reverted to avoid confusion?

Reply:
oh yea sorry for the confusion